### PR TITLE
Add more personal story to preface

### PR DESCRIPTION
**Pull request by rikhuijzer**
I was thinking yesterday about Statistical Rethinking and remembered that I could easily remember **why** McElreath wrote the book. He wished that he was taught statistics like he did in the book.

For our preface, I now also added a bit more personal viewpoint.

Please both of you check whether you agree with my opinion.

### CODE REVIEW

File: contents/preface.md [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+We discuss some of our arguments in @sec:why_julia.
+Compared to other languages, Julia is one of the newest languages around.
+This means that the ecosystem around the language is sometimes difficult to navigate through.
+It's difficult to figure out where to start and how all the different packages fit together.
+That is why we decided to create this book!
+We wanted to make it easier for researchers, and especially our colleagues, to start using this awesome language.
+
+Like discussed above, each language has it's strengths and weaknesses.
+In our opinion, data science is definitely a strength of Julia.
+At the same time, all three of us need data science in our day to day life.
```
Comment (storopoli) [inline contents/preface.md:16]: need is too much. What about use?

Also "need data science **tools**"

File: contents/preface.md [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+And, probably, you need data science too!
```
Comment (storopoli) [inline contents/preface.md:17]: same as above (use / tools)

File: contents/preface.md [hunk 3]
```diff
@@ -1,9 +1,22 @@
...
+That is why this book has a focus on data science.
+Next, we emphasize the **"data" part of data science** and why data skills are, and will remain, in **high demand** in industry as well as in academia.
```
Comment (storopoli) [inline contents/preface.md:19]: Next what?

File: contents/preface.md [hunk 4]
```diff
@@ -1,9 +1,22 @@
...
 
 Second, we make an argument for **incorporate software engineering practices into data science** which should reduce friction in updating and sharing code with collaborators.
```
Comment (storopoli) [inline contents/preface.md:21]: Where is the first to have a second?

**Review by storopoli — COMMENTED** [review]
I think this is a great intro to the preface. Awesome! Here are some suggestions.

**Comment by rikhuijzer** [comment]
@lazarusA, can you also take a look?

File: contents/preface.md [hunk 5]
```diff
@@ -1,11 +1,24 @@
 # Preface {#sec:preface}
 
-In this book, we discuss two important aspects of data analysis.
-Actually, there is a third one, Julia, but it deserves a whole chapter (see @sec:why_julia).
-
-First, we emphasize the **"data" part of data science** and why data skills are, and will remain, in **high demand** in industry as well as in academia.
-
-Second, we make an argument for **incorporate software engineering practices into data science** which should reduce friction in updating and sharing code with collaborators.
+There are many programming languages and each and every one has its strengths and weaknesses.
```
Comment (lazarusA) [inline contents/preface.md:3]:
```suggestion
There are many programming languages and each and every one of them has its strengths and weaknesses.
```

File: contents/preface.md [hunk 6]
```diff
@@ -1,11 +1,24 @@
...
+Some languages are very quick, but verbose.
+Other languages are very easy to write in, but slow.
```
Comment (lazarusA) [inline contents/preface.md:5]:
```suggestion
Other languages are very easy to write in, but slow. This is known as the `two-language` problem and `Julia` aims at circumventing this problem. 
```

**Review by lazarusA — COMMENTED** [review]
Looks nice.

**Pull request merged**